src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx

```diff
@@ -848,6 +848,7 @@ The following features are not available for the .NET agent:
 
 * The .NET agent does not support [trim self-contained deployments and executables](https://docs.microsoft.com/en-us/dotnet/core/deploying/trim-self-contained), because the compiler can potentially trim assemblies that the agent depends on.
 * [Infinite Tracing](/docs/distributed-tracing/infinite-tracing/introduction-infinite-tracing) is not supported on Alpine Linux due to a [GRPC compatibility issue](https://github.com/grpc/grpc/issues/21446). See [this agent issue](https://github.com/newrelic/newrelic-dotnet-agent/issues/289) for more information.
+* The .NET agent does not support [Native Ahead of Time (AOT) deployment](https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/?tabs=net8plus%2Cwindows) for .NET applications because just-in-time (JIT) compilation is required for the agent to function properly. 
```

Comment:
```suggestion
* The .NET agent doesn't support [Native Ahead of Time (AOT) deployment](https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/?tabs=net8plus%2Cwindows) for .NET applications because just-in-time (JIT) compilation is required for the agent to function properly. 
```